src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetLocationsMappingPage.tsx

```diff
@@ -100,11 +115,27 @@ export default function ReleaseApiDataSetLocationsMappingPage() {
 
   const navItems: NavItem[] = useMemo(() => {
     return [
+      ...(Object.keys(deletedLocationGroups).length > 0
```

Comment:
I didn't add the filter column equivalents for this and new location groups to the nav on the filters mapping page, but it probably makes sense to add them in to make it consistent with the locations page.